Tests/PackageLoadingTests/PackageBuilderTests.swift

```diff
@@ -2892,6 +2895,43 @@ class PackageBuilderTests: XCTestCase {
         }
     }
     
+    func testXcodeResources5_5AndLater() throws {
+        // In SwiftTools 5.5 and later, xcbuild file types are only supported when explicitly passed via additionalFileRules.
+        
+        let root: AbsolutePath = "/Foo"
+        let Foo = root.appending(components: "Sources", "Foo")
```

Comment:
nit: casing

Reply:
Here I was matching the existing style in `testXcodeResources` (renamed to `testXcodeResources5_4AndEarlier`). Should I change both to use `foo` instead?

Reply:
yes please